src/ingestion-server/kafka-s3-connector/iam.ts

```diff
@@ -131,6 +131,7 @@ export function createRoleForS3SinkConnectorCustomResourceLambda(
         'kafkaconnect:ListCustomPlugins',
         'kafkaconnect:DeleteCustomPlugin',
         'kafkaconnect:UpdateConnector',
+        'kafkaconnect:TagResource',
```

Comment:
Add deleteTag and updateTag if available too

Reply:
There is no deleteTag and updateTag
Add ListTagsForResource and UntagResource actions permission.